docs/proposals/03-loadbalancer.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Network Loadbalancer
+
+oep-number: 3
+
+creation-date: 2022-10-18
+
+status: implementable
+
+authors:
+
+- "@gehoern"
+reviewers:
+- "@adracus"
+- "@afritzler"
+- "@guvenc"
+- "@MalteJ"
+
+---
+
+# OEP-3: Network Loadbalancer
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+    - [Details](#details)
+- [Proposal](#proposal)
+
+## Summary
+Network service for Load Balancing is needed in every modern infrastructure (e.g. Kubernetes). Load balancing makes services scalable, fault-tolerant and accessible from external users. The load balancer described here is a L3-Loadbalancer (so it implements an IP level load balancing). The load balancer function is a standard network function that can be to be deployed (as an implementation detail) on a platform/node that is capable of running network functions (e.g. dp-service enabled nodes/nic's) 
```

Comment:
We should omit the part about how load balancers might be deployed, especially mentioning `dp-service`.

Reply:
done